logic/block.js

```diff
@@ -191,11 +191,7 @@ class Block {
 
 		try {
 			const data = this.getBytes(block);
-			const dataWithoutSignature = Buffer.alloc(data.length - signatureLength);
-
-			for (let i = 0; i < dataWithoutSignature.length; i++) {
-				dataWithoutSignature[i] = data[i];
-			}
+			const dataWithoutSignature = data.slice(0, -signatureLength);
```

Comment:
Please, we feel this commit changes critical part of the code without adding any readability. At this point of heavy development, we prefer to be cautious and ask you kindly to revert or delete the related commit.

Reply:
You are talking about https://github.com/LiskHQ/lisk/pull/1874/commits/950ddcbd73aacf1035200aca66accb6df00caad8, right?